packages/glob/src/internal-glob-options.ts

```diff
@@ -36,4 +36,11 @@ export interface GlobOptions {
    * @default true
    */
   omitBrokenSymbolicLinks?: boolean
+
+  /**
+   * Indicates whether to include hidden files (files and directories starting with a `.`).
+   * 
+   * @default false
```

Comment:
Don't we want the default behavior to be to exclude?

Reply:
We want the default to be exclude for the action that consumes this library, I don't think we need or should make a breaking change in this library.

Reply:
Makes sense for changes to the glob package to be backward compatible but just curious, what other actions use this package? Are there any actions other than upload-artifact where we would want to pass true?

Reply:
nit: the comment should read "Indicates whether to exclude hidden files.."

Reply:
> Are there any actions other than upload-artifact where we would want to pass true?

https://github.com/actions/toolkit/network/dependents?owner=actions&dependent_type=REPOSITORY&owner=actions

I see `actions/deploy-pages`, `actions/upload-artifact`, and `actions/download-artifact`.